Sources/Transcripts/Collections/Tours.swift

```diff
@@ -64,6 +50,21 @@ extension Episode.Collection {
         title: "www.pointfree.co",
         whereToGoFromHere: nil
       ),
+      .init(
+        blurb: #"""
+        We open sourced the [Composable Architecture](http://github.com/pointfreeco/swift-composable-architecture) after many, _many_ months of developing the concepts from first principles in Point-Free episodes. In this tour we build a small application from scratch, focus on breaking it down to into small understandable units, and write a full test suite to exercise every subtle aspect of the application.
+        """#,
+        coreLessons: [
+          .init(episode: .ep100_ATourOfTheComposableArchitecture_pt1),
+          .init(episode: .ep101_ATourOfTheComposableArchitecture_pt2),
+          .init(episode: .ep102_ATourOfTheComposableArchitecture_pt3),
+          .init(episode: .ep103_ATourOfTheComposableArchitecture_pt4),
+        ],
+        isHidden: true,
```

Comment:
this just moved below. it's the 0.x tour and it's hidden but kept around to preserve links.